Templates/Applications/Salesforce.json

```diff
@@ -16,6 +16,10 @@
         "selectGroupsAllowed": false
       }
     },
+    "applicationType": {
+      "name": "SAML"
+    },
```

Comment:
Let's get rid of the extra object here.  I don't think this needs to be more complicated than necessary.